src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/generate_static_shapes.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/node.hpp"
+
+namespace ov {
+namespace test {
+namespace utils {
+
+using StaticShapesMap = std::map<ov::NodeTypeInfo, std::function<ov::Shape(
+        const std::shared_ptr<ov::Node>& node,
+        size_t port,
+        uint64_t dimMin,
+        uint64_t dimMax,
+        int seed
```

Comment:
Minor: `seed` is potentially no so important arg

Reply:
I have to introduce parameter `seed` to generate new static shapes at each call of `generate()`.
Otherwise 3 calls of `generate()` with the same parameters without `seed` return 3 identical static shapes.

Reply:
We need to validate dynamic shapes using several different static shapes for inference.
We decided to use 2 corner cases: min and max static shapes and some shape between min and max.
In some cases, max_shapes can be enormous, so inference will take a lot of time in this case. It's the main reason, why we decided to use char limitation to generate max shape.

So, As I can see you generate pseudo random static shapes from [dimMin, dimMax) in the PR. What do you think about to merge previos logic to generator and generate 3 static shapes in one call (min, max, mid)?
In this case, we will have:
```
inputShapes.push_back(InputShape{param->get_partial_shape(), it->second(node, port_id);});
```

Reply:
Restored the generation of minimum, maximum and random values.
Moreover, the generation function now returns `InputShape` of 3 values  instead of a single `Shape`  (https://github.com/openvinotoolkit/openvino/pull/24125/commits/dec8fc97d918ef0267d1b21c877f53de6e9d1735)